keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -275,6 +275,8 @@ The `ClusterClaim` resource provides a way to store identification related, clus
 
 ### Well known claims
 
+The `ClusterClaim` CRD will support two specific claims under the well known names `id.k8s.io` and `clusterset.k8s.io`. Being "well known" means that they must conform to the requirements described below, and therefore can be dependent on by multi-cluster implementations to achieve use cases dependent on knowlege of a cluster's ID or ClusterSet membership.
```

Comment:
nit: s/dependent/depended/ in first case

Reply:
Fixed in ab78752